FizzBuzz.py

```diff
@@ -1,27 +1,42 @@
-from types import ListType, IntType
+"""Print all integers from 1 to 100. However, for multiples of three,
+print 'Fizz' instead of the number, for multiples of five, print
+'Buzz', and for numbers which are multiples of both three and five,
+print 'FizzBuzz'.
+"""
+def is_appropriate(number):
+    """Assert whether or not a given number is an integer between 0 and
+    101. Is so, return the boolean True.
+    """
+    assert isinstance(number, int)
+    assert 0 < number <= 100
+    return True
 
-def create_list(size):
-    #takes one positive int 'size' and returns a list of all integers 0 to 'size'
-    assert isinstance(size, IntType)
-    some_list = []
-    for i in range(0, size+1):
-        some_list.append(i)
-    return some_list
+def determine_element(element):
+    """Determine whether the element should be named, 'Fizz', 'Buzz',
+    'FizzBuzz', or itself, make the change to the element(if
+    applicable), and return it.
+    """
+    if is_appropriate(element):
+        if element % 15 == 0:
+            return 'FizzBuzz'
+        elif element % 3 == 0:
+            return 'Fizz'
+        elif element % 5 == 0:
+            return 'Buzz'
+        return element
 
-def insert_something(some_list, some_thing, divisor):
-    #takes a list 'someList', a desired variable to be inserted into the list 'someThing', and an
-    #int 'divisor'. 'someThing' gets inserted into 'someList' at every element where the element's
-    #index is divisible by 'divisor'
-    assert isinstance(some_list, ListType)
-    assert isinstance(divisor, IntType)
-    for index, content in enumerate(some_list):
-        if index%divisor == 0 and index != 0:
-            some_list[index] = some_thing
-    return some_list
+def format_element(number, element):
+    """Structure the number according to the amount of digits it
+    contains and print it out with its corresponding element.
+    """
+    if is_appropriate(number):
+        if number < 10:
+            return "  " + str(number) + "  :  "+str(element)
```

Comment:
http://stackoverflow.com/questions/1402048/how-do-i-right-align-numeric-data-in-python

you don't need to right align the numbers but you probably chose this as a stretch target. try using the advice in stack overflow as well as string.format to get your desired effect